src/awsexplorer/activation.ts

```diff
@@ -179,6 +182,13 @@ async function registerAwsExplorerCommands(
         )
     )
 
+    context.subscriptions.push(
+        vscode.commands.registerCommand(
+            'aws.copyArn',
+            async (node: AWSResourceNode) => await copyArnToClipboard(node.getArn())
```

Comment:
The generalized nature of commands, combined with the less-granular-than-ideal "when" clauses associated with them mean its easy to get unintended data passed into these commands.

Please consider adding the following validation checks:
* node is not undefined
* node is an instanceof AWSResourceNode
* try/catch error handling, in case an unexpected node object still gets past all of the above, and doesn't contain "getArn"